ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml

```diff
@@ -63,16 +63,16 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/serial
     workflow: hypershift-aws-conformance
-- as: conformance-aws-ovn-4-12
+- as: conformance-aws-ovn-4.12
   cron: 0 */6 * * *
   steps:
     workflow: hypershift-aws-conformance
-- as: conformance-azure-ovn-4-12
+- as: conformance-azure-ovn-4.12
   cron: 0 */6 * * *
   steps:
     cluster_profile: azure4
     workflow: hypershift-azure-conformance
-- as: conformance-proxy-aws-ovn-4-12
+- as: conformance-proxy-aws-ovn-4.12
```

Comment:
Could we tighten this up just a little more in case there are other lurking assumptions that will go bad.

As an example could we get closer to: periodic-ci-openshift-release-master-ci-4.12-e2e-aws-sdn

Maybe like 4.12-conformance-aws-ovn-proxy?